### ISSUE TITLE
'Filter Lists' preferences panel

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#1792]
Unclicking one of the Essentials causes it to disappear from the Essentials (after refresh). Instead, it should stay in the Essentials category and simply be unselected. 

The "uBlock Filters - Annoyances" list moves down to Annoyances when unselected (see below), however, in the case of "uBlock Filters" or "uBlock Filters - Privacy" the list seems to disappear completely and there is no way to re-enable it in the UI.

Basically lists should stay in their categories, whether selected or not.

![image](https://user-images.githubusercontent.com/737638/113559484-074ec280-9634-11eb-9bd0-307b5dc3e079.png)

**Comment by dhowe** [comment]
Ok. How about the case I mention where disabling a list makes it disappear completely and it is then impossible to re-enable?

**Comment by mneunomne** [comment]
Unless you are experiencing smth that I cannot reproduce I would say you need to click the "+" sign for the unchecked options to appear (which I do think is the case given your previous screenshot).

![Screen Shot 2021-04-09 at 01 18 11](https://user-images.githubusercontent.com/4967860/114107589-975d6800-98d1-11eb-8b5f-421fcf6c28dd.png)
![Screen Shot 2021-04-09 at 01 18 08](https://user-images.githubusercontent.com/4967860/114107593-988e9500-98d1-11eb-80ec-e09f2dd528e3.png)

**Comment by dhowe** [comment]
wow, that is quite confusing for the user... but you're right, it is the same as ublock
so lets leave it for now
